src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -30,37 +30,46 @@ class RawlsApiServiceActor(val workspaceServiceConstructor: UserInfo => Workspac
   def apiRoutes = options{ complete(OK) } ~ baseRoute ~ workspaceRoutes ~ entityRoutes ~ methodConfigRoutes ~ submissionRoutes ~ adminRoutes ~ userRoutes
   def registerRoutes = options{ complete(OK) } ~ createUserRoute ~ getUserStatusRoute
 
+  def receive = runRoute(
```

Comment:
This diff is a little confusing because I reformatted the runRoute block and moved swaggerRoute into RootRawlsApiService trait. Other than that, I added versionRoute.